sites/main-site/src/content/blog/2024/maintainers-minutes-2024-07-26.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: 'Maintainers Minutes: August 2024'
+subtitle: 'Keeping you informed of the latest maintainers discussions'
+pubDate: 2024-08-14T12:00:00+02:00
+headerImage: '/assets/images/blog/maintainers-minutes-2024-07-26/maintainers_duck.png'
```

Comment:
This is huge when rendering on the website, can we limit the size of the header @mashehu ?

Reply:
Done. there is now a headerImageDim option to control that.